packages/next-intl/test/middleware/middleware.test.tsx

```diff
@@ -68,6 +68,33 @@ beforeEach(() => {
   vi.clearAllMocks();
 });
 
+it.only('has docs that suggest a reasonable default matcher', () => {
+  const regexStringFromDocs = '/((?!api|_next|.*\\..).*)';
```

Comment:
I just tried this regex in the codesandbox repro but I see the problem still occurs

Reply:
Yep, please see https://github.com/amannn/next-intl/pull/505#discussion_r1325700738